tidb-binlog/bidirectional-replication-between-tidb-clusters.md

```diff
@@ -9,9 +9,8 @@ aliases: ['/docs/dev/tidb-binlog/bidirectional-replication-between-tidb-clusters
 > **Warning:**
 >
 > - Currently, bidirectional replication is still an experimental feature. It is **NOT** recommended to use it in the production environment.
```

Comment:
This specific line should be removed due to the improvements in FD-1689. I am not sure if this is tracked in a different PR?

Reply:
It seems that the line was ignored in https://github.com/pingcap/docs/pull/18568. I'll follow up.

Reply:
This line was removed in https://github.com/pingcap/docs/pull/18606.